### DISCUSSION TITLE
Using with pusher

### DISCUSSION THREAD

**Discussion by alalfakawma** [spatie/laravel-activitylog#782] [Q&A] [answered]
Hello, I'm trying to use this with Pusher. And I have an ActivityLogged event, which when triggered will update the frontend notification via Pusher. But the thing is, what would be the best way to trigger the ActivityLogged event? I mean, I could go to my model observers and fire the event one by one. But is there a better way? I mean, does this package expose an event? Like activity logged or something, which we could listen to and use that to fire other events.

**Comment by Gummibeer** [comment, marked as answer]
Hey,

you have already listed the "best" way yourself. 😉
The `Activity` model is a default eloquent model - so you can add an observer on this model which will trigger every time any activity is created.

**Reply by alalfakawma** [reply]
Oh wow. LOL. I LOL'ed hard. How did I miss something so obvious. Maybe I was overthinking it. Thank you so much for pointing that out for me, it really bugged me yesterday. :rofl:
